translationCoreApps/ScripturePane
# Update Components to use new BCV reducers #1816

## This PR Addresses:

Update Components to use new BCV reducers from #1816

## To Test:

Load and ensure Scripture Pane now works with new BCV changes.

- Test all bibles
- Test UGNT functionality
- Test Chapter Modal view
- Test Target Languages


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/scripturepane/54)
